kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -28,26 +28,43 @@ def airflow_commands():
 @click.option("-e", "--env", default="local")
 @click.option(
     "-t",
-    "--target-dir",
+    "--target-path",
     "target_path",
-    type=click.Path(writable=True, resolve_path=True, file_okay=False),
-    default="./airflow_dags/",
+    default="./airflow_dags/{package_name}_dag.py",
+)
+@click.option(
+    "-j",
+    "--jinja-file",
+    "jinja_template",
+    type=click.Path(exists=True, readable=True, resolve_path=True, file_okay=True),
```

Comment:
```suggestion
    type=click.Path(exists=True, readable=True, resolve_path=True, file_okay=True, dir_okay=False),
```
And then you should be able to get rid of the `if jinja_template.is_file` conditional also, since it will already be checked by `click.Path`.